src/chrome/content/rules/CafePress.xml

```diff
@@ -34,8 +34,8 @@ Fetch error: http://server.iad.liveperson.net/ => http://server.iad.liveperson.n
 		to="https://content.cafepress.com/"/>
 
 	<!--	rewriting help to liveperson breaks links under "design your own"
-		in the footer.  Fix this by rewriting back.	-->
-	<rule from="^https://server\.iad\.liveperson\.net/make/"
-		to="https://www.cafepress.com/make/"/>
```

Comment:
I think you can completely remove this rule and the related target because it's already handled in the (currently `default_off`) [`LivePerson.net.xml`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/LivePerson.net.xml) ruleset. It will also fix the issue where you have a target without a related rule.

Reply:
Thank you for pointing out the related ruleset